docs/md/dev-process.md

```diff
@@ -131,6 +139,59 @@ Here are some tips for WFL development.
 Some of this advice might help when testing Liquibase migration or other
 changes that affect WFL's Postgres database.
 
+### setting up a local Postgres
+
+You can test against a local Postgres before running Liquibase or SQL against a
+shared database in `gotc-dev` or *gasp* production.
+
+1. Install Postgres locally.
+    You need version 11 because that is what Google's hosted service supports,
+    and there are differences in the SQL syntax.
+
+    ```
+    brew install postgresql@11
+    ```
+
+2. Start Postgres.
+
+    ```
+    pg_ctl -D /usr/local/var/postgresql@11 start
+    ```
+    !!! tip
+        It might be useful to set up some an alias for postgres if you are using
+        zsh, for example:
+        ```
+        alias pq="pg_ctl -D /usr/local/var/postgresql@11"
+        ```
+        thus you could use `pq start` or `pq stop` to easily spin up and turn down
+        the db.
+
+3. [Optional] Create wfl DB.
+
+    If you see errors like this when launching a local WFL server
+    or applying liquibase updates:
+
+    ```
+    FATAL:  database "wfl" does not exist
+    ```
+
+    You should do as instructed within your terminal:
+
+    ```
+    createdb wfl
+    ```
```

Comment:
nit: might be helpful to also include `dropdb wfl` here in case folks want to recreate the local DBs?

Reply:
Added.